packages/core/src/Flex/README.mdx

```diff
@@ -0,0 +1,107 @@
+---
+title: 'Flex'
+type: 'component'
+slug: /components/flex/
+github: 'https://github.com/contentful/forma-36/tree/master/packages/forma-36-react-components/src/components/Flex'
+storybook: 'https://f36-storybook.contentful.com/?path=/story/components-flex--default'
+---
+
+Flex components is [Box] with `display: flex` and abilities to apply other flexbox CSS properties via properties of the component.
+
+## How to use Flex
+
+The Flex component is useful because it helps with several common cases:
+
+- Create a one direction layout (vertical or horizontal) with uses Forma 36 spacing.
+- Compose new component and allow to override element using the `as` property.
+
+### Import
+
+```js static=true
+import { Flex } from '@contentful/f36-components';
+```
+
+## Code examples
+
+### Margins & Paddings
+
+Flex components inherits behaviour of [Box] component and allows to pass margins and paddings that are based on our [spacing system].
+
+To control the behaviour of the Flex component you can use specific properies that match [all main Flexbox related CSS attributes](https://cssreference.io/flexbox/).
+
+```js
+() => {
+  const ColorBox = ({ backgroundColor, ...rest }) => (
+    <Flex
+      as="div"
+      justifyContent="center"
+      alignItems="center"
+      style={{
+        width: '120px',
+        height: '120px',
+        color: tokens.colorWhite,
+        backgroundColor,
+      }}
+      {...rest}
+    />
+  );
+
+  return (
+    <Flex
+      justifyContent="space-between"
+      alignItems="center"
+      marginBottom="spacingM"
+    >
+      <ColorBox backgroundColor={tokens.colorGreenBase}>Example item</ColorBox>
+      <ColorBox backgroundColor={tokens.colorRedBase}>Example item</ColorBox>
+      <ColorBox backgroundColor={tokens.colorBlueBase}>Example item</ColorBox>
+    </Flex>
+  );
+};
+```
+
+### Using Flex.Spacer
+
+Flex.Spacer allows you to create an adjustable, empty space that can be used to tune the spacing between child elements within Flex.
+
+```jsx
+<Flex>
+  <Box
+    padding="spacingM"
+    style={{ backgroundColor: tokens.colorRedBase, color: tokens.colorWhite }}
+  >
+    Box 1
+  </Box>
+  <Flex.Spacer />
+  <Box
+    padding="spacingM"
+    style={{ backgroundColor: tokens.colorGreenBase, color: tokens.colorWhite }}
+  >
+    Box 2
+  </Box>
+</Flex>
+```
+
+### Real-world example
+
+```jsx
+<Flex alignItems="center">
+  <Box>
+    <Subheading>User name</Subheading>
+  </Box>
+  <Flex.Spacer />
```

Comment:
Would `Flex.Spacer` create an empty element in the DOM?
And if yes, is this ok? Is it a11y? Do we want to recommend that?

I'm asking because it seems that we can achieve the same effect of the spacer here by just passing a marginBottom to the Box

Reply:
valid point, I'm going to remove Flex.Spacer for now